click_odoo_contrib/restoredb.py

```diff
@@ -29,7 +30,15 @@ def _restore_from_folder(dbname, backup, copy=True, jobs=1, neutralize=False):
 
     odoo.service.db._create_empty_database(dbname)
     pg_args = ["--jobs", str(jobs), "--dbname", dbname, "--no-owner", dbdump_file_path]
-    if odoo.tools.exec_pg_command("pg_restore", *pg_args):
+    pg_env = odoo.tools.misc.exec_pg_environ()
+    r = subprocess.run(
+        "pg_restore",
+        pg_args,
```

Comment:
```suggestion
        ["pg_restore", *pg_args],
```

This will hopefully make tests pass.